from python_utilities.exceptions.python_utility_error import PythonUtilityError

class PythonUtilityJSONError( PythonUtilityError ):

    pass

#-- END class PythonUtilityJSONError --#
